src/main/java/salesforce/salesforceapp/entities/opportunities/Oppy.java

```diff
@@ -32,7 +32,7 @@ public String getId() {
 //      APIOppy apiOppy = new APIOppy(this);
 //      id = apiOppy.getSObjectRecordFieldValueByField("id", "Name", oppyName);
 //    }
-    setId("0061I000007UZAUQA4");
+    setId("0061I000007UZSFQA4");
```

Comment:
Why are you setting the id hard coded?

Reply:
Ia am sorry.I could't parse the json response of creating an Oppy.